### ISSUE TITLE
New Confirmations for Spain and France comparing 04/30 and 05/01 Daily Reports

### ISSUE THREAD

**Issue by maisonml** [CSSEGISandData/COVID-19#2457]
Comparing csse_covid_19_daily_reports/05-01-2020.csv and csse_covid_19_daily_reports/04-30-2020.csv, new confirmations for Spain and France are shown as 0 and 6 respectively. Please check this. 

Country_Region TotalConfirmed  NewConfirmations
        Spain         213435                0
       France         167305                6

**Comment by valeriupredoi** [comment]
good call! was about to open an issue myself :beer:

**Comment by joaopedrosantos** [comment]
And -23k confirmed cases yesterday. Weird

Country	Date	Confirmed_Cases_Increase
Spain	30-04-2020	-23.464
Spain	01-05-2020	0

**Comment by dcleblanc** [comment]
Yes, the feed for Spain seems broken - Worldometer, and other sources are still updating.

**Comment by StateCenterKid** [comment]
Spain is still broken.  Correction to Worldometer values in R goes something like this:

Covid_Data_State$Confirmed[(Covid_Data_State$Country.Region=="Spain")   &is.na(Covid_Data_State$Province.State)&Covid_Data_State$ReportTime>=as.POSIXct("2020-04-29 01:00:00") &Covid_Data_State$ReportTime<=as.POSIXct("2020-05-03 11:00:00")]=c(236899,239340,242979,245589,247122)

**Comment by ThEls-1847** [comment]
Values (Infected, deaths and recoveries) for Spain have not changed on May, 1st. (compared to April, 30th.) on May, 2nd, values appear to include the increase from April, 30th and May, 1st.

**Comment by DanDefATG** [comment]
Is there any news on these countries data?
